samples/SampleApp/Properties/serviceDependencies.local.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "storage1": {
+      "type": "storage.emulator",
+      "connectionId": "AzureWebJobsStorage"
+    }
+  }
+}
```

Comment:
What are these? Did these just get added by VS?

Reply:
I am not sure tbh. Even our `FunctionApp` sample has that. It might have been because I copied from there. Let me remove them from here and see.